web/src/test/java/com/epam/rft/atsy/cucumber/application/NewApplicationPopupStepDefs.java

```diff
@@ -0,0 +1,217 @@
+  private static final String DIRECT_IN_HUN = "direkt";
+  private static final String COMMENT_IN_HUN = "Megjegyzés";
+  private static final String NEW_APPLY_IN_HUN = "Új jelentkezés";
+
+  private static final String POSITION_DROP_DOWN_ERROR_MESSAGE_IN_HUN = "Kérem válasszon egy pozíciót";
+  private static final String APPLICATION_SOURCE_DROP_DOWN_ERROR_MESSAGE_IN_HUN = "Kérem válasszon egy jelentkezési forrást";
+
+  private static final Integer COLUMN_ZEROTH = 0;
+  private static final Integer COLUMN_FIRST = 1;
+  private static final Integer COLUMN_SECOND = 2;
+  private static final Integer COLUMN_THIRD = 3;
+
+
+  @Given("^the user is on a Candidate A profile page$")
+  public void the_user_is_on_a_candidate_A_profile_page() throws Throwable {
+    webDriver.get("http://localhost:8080/atsy/secure/candidate/1");
+  }
+
+  @When("^the user clicks on the Új jelentkezés hozzáadása button$")
+  public void the_user_clicks_on_the_add_application_button() throws Throwable {
+    webDriver.findElement(By.id(ADD_APPLICATION_BUTTON)).click();
+  }
+
+  @Then("^the Application page appears$")
+  public void the_application_page_appears$() throws Throwable {
+    assertThat(webDriver.getCurrentUrl(), equalTo("http://localhost:8080/atsy/secure/application?candidateId=1"));
+  }
+
+  @And("^the New application popup appears$")
+  public void the_new_application_popup_appears() throws Throwable {
+    assertThat(webDriver.findElement(By.id(POP_UP_CONTENT)).isDisplayed(), is(true));
+  }
+
+  @And("^the user can select a position from a dropdown$")
+  public void the_user_can_select_a_position_from_a_dropdown() throws Throwable {
+    Integer dropdownSize = getOptionsSizeFromDropdown(POSITION);
+    assertThat(dropdownSize, greaterThan(0));
+  }
+
+  @And("^the user can select a source of application from the list$")
+  public void the_user_can_select_a_source_of_application_from_the_list() throws Throwable {
+    Integer dropdownSize = getOptionsSizeFromDropdown(CHANNEL);
+    assertThat(dropdownSize, greaterThan(0));
+  }
+
+  @And("^the user can add a comment$")
+  public void the_user_can_add_a_comment() throws Throwable {
+    assertThat(webDriver.findElement(By.id(DESCRIPTION)).isEnabled(), is(true));
+  }
+
+  @And("^the user can submit the new application$")
+  public void the_user_can_submit_the_new_application() throws Throwable {
+    assertThat(webDriver.findElement(By.id(SAVE_NEW_APPLY_BUTTON)).isEnabled(), is(true));
+  }
+
+
+  @Given("^the user is on a New application popup of an existing candidate$")
+  public void the_user_is_on_a_new_application_popup_of_an_existing_candidates() throws Throwable {
+    webDriver.get("http://localhost:8080/atsy/secure/application?candidateId=1");
+  }
+
+  @And("^position Fejlesztő exists$")
+  public void position_developer_exists() throws Throwable {
+    List<String> existPositionNameList = convertWebElementsToStringList(webDriver.findElements(By.id(POSITION)));
```

Comment:
On my machine this returns a single element where the options are concatenated by CRLF.

As a result the scenarios with this step are not passing on my computer.